### DISCUSSION TITLE
RichTextEditor Wrap Action Needs Name?

### DISCUSSION THREAD

**Discussion by raywong-fb** [GrapesJS/grapesjs#4511] [Q&A]
I'm not sure if this is by design, or possibly a bug. But I noticed I wasn't able to programmatically `remove` the default `wrap` action (ie. `rte.remove('wrap')`) of the RTE and it looks like it's due to the model does not include a `name` attribute [here](https://github.com/artf/grapesjs/blob/dev/src/rich_text_editor/model/RichTextEditor.js#L70-L85).

Is this a bug or is there a reason there isn't a `name` associated with the wrap action? I was able to get around this by passing an array in the init config that leaves out `wrap`, like:

```
richTextEditor: {
  actions: ["bold", "italic", "underline", "link"],
},
```

**Comment by artf** [comment]
Thanks @raywong-fb definitely a mistake for not including the name, I'll update it.
